charts/fybrik/templates/openmetadata-connector/openmetadata-connector-rbac.yaml

```diff
@@ -0,0 +1,26 @@
+{{- $autoFlag := and .Values.coordinator.enabled (eq .Values.coordinator.catalog "openmetadata") }}
```

Comment:
Why is RBAC required? It's just a connector that contacts the server and gets back the data. Same for editor-role.

Reply:
removed